deploy/charts/nfspv-provisioner/templates/NOTES.txt

```diff
@@ -0,0 +1,12 @@
+Thank you for installing {{ .Chart.Name }} 😀
+
+Your release is named {{ .Release.Name }} and it's installed to namespace: {{ .Release.Namespace }}.
+
+The OpenEBS NFSPV Provisioner has been installed check its status by running:
+$ kubectl get pods -n {{ .Release.Namespace }}
+
+Use `kubectl get bd -n {{ .Release.Namespace }} ` to see the list of
```

Comment:
This command does not seem relevant to this chart.

Reply:
Thanks mate , removed it

Reply:
Removed the mentioned line, thanks